openfpga_flow/scripts/run_fpga_flow.py

```diff
@@ -489,6 +489,21 @@ def run_yosys_with_abc():
         "LUT_SIZE": lut_size,
         "OUTPUT_BLIF": args.top_module+"_yosys_out.blif",
     }
+
+    for indx in range(0, len(OpenFPGAArgs), 2):
+        tmpVar = OpenFPGAArgs[indx][2:].upper()
+        ys_params[tmpVar] = OpenFPGAArgs[indx+1]
+    
+    if 'YOSYS_FAMILY' not in ys_params.keys():
+        # define default family as 'qlf_k4n8'
+        ys_params['YOSYS_FAMILY'] = "qlf_k4n8"
+
+    # prefix value of YOSYS_MODE with '-' as an option in yosys script
+    if 'YOSYS_MODE' in ys_params.keys():
+        ys_params['YOSYS_MODE'] = "-" + ys_params['YOSYS_MODE']
```

Comment:
You may not need to add '-' in the script. You can simply add the '-' in task configuration file.